runtime/ftplugin/qf.vim

```diff
@@ -14,3 +14,10 @@ let b:undo_ftplugin = "set stl<"
 
 " Display the command that produced the list in the quickfix window:
 setlocal stl=%t%{exists('w:quickfix_title')?\ '\ '.w:quickfix_title\ :\ ''}\ %=%-15(%l,%c%V%)\ %P
+
+autocmd Syntax <buffer>
+      \ if exists('w:quickfix_title') && w:quickfix_title =~# '^:\%(Man\|Help\) TOC$' |
+      \   syntax match qfHideTOC "^.*| " conceal containedin=ALL |
+      \   setlocal concealcursor=nvi |
```

Comment:
Why not set this to `nvic`? Doesn't make sense for the file name to pop up if I search the list.